### ISSUE TITLE
Add Radial Basis Function

### ISSUE THREAD

**Issue by wilsonrljr** [wilsonrljr/sysidentpy#99]
One of the key features of SysIdentPy is its ability to support different types of basis functions for system identification. Currently, SysIdentPy supports both Polynomial and Fourier Basis Function, which are commonly used in many system identification applications.

However, there is another widely used basis function that is not yet supported by SysIdentPy, and that is the Radial Basis Function (RBF). The RBF is a well-known basis function used for system identification, which has been proven to be effective in many applications. In fact, there are several research papers that have used NARMAX models with RBF, and have shown that it can lead to accurate and efficient system identification.

To ensure that SysIdentPy remains at the forefront of system identification research, the development team has decided to help a interested person to implement  RBF as a new option for basis functions in SysIdentPy. This new feature will enable users to use RBF in their system identification tasks, which will help them to achieve more accurate and reliable results.

The implementation of RBF in SysIdentPy will involve multiple steps, including the development of new algorithms and methods to support this basis function, as well as the integration of the necessary libraries and dependencies. This will require some work, but the SysIdentPy maintainer (wilsonrljr) is committed to helping in all steps of the implementation to make RBF available for the users.

In conclusion, the addition of RBF as a new option for basis functions in SysIdentPy is a significant step forward for the library, and will enable users to take advantage of one of the most widely used basis functions in system identification. By providing this new feature, SysIdentPy will continue to be a valuable tool for researchers and engineers who are working on complex system identification problems. If you are interested in learning more about how RBF is used in NARMAX models, we recommend checking out the paper linked above for a more in-depth explanation.

https://eprints.soton.ac.uk/251141/1/777327803_content.pdf

**Comment by nataliakeles** [comment]
I really want to work on this issue!
